{"name": "test_Get_target_device_info", "status": "broken", "statusDetails": {"message": "KeyError: 0", "trace": "self = <Testcase_Devices.test_04_Get_target_device_info.Test_04_Get_target_device_info object at 0x0000022711F8F5E0>\n\n    @allure.feature('【需求点】：A004-获取固定SN设备的信息')\n    @allure.story('【正常获取】')\n    @allure.severity('normal')\n    def test_Get_target_device_info(self):\n        ip = ReadIP().Read()\n        urls = ReadURL().Read()\n        device_sn = ReadSN().Read()\n        url1 = urls['url04']['url']\n        url = ip + url1\n        url = url.replace('sn',device_sn)\n        auth = ReadAuthorization().Read()\n        headers = {\n            'Authorization': auth,\n        }\n        method = 'get'\n        # 发送请求\n        res = RequestTools().send_requests(method=method,url=url,header=headers)\n        print(\"-----------------\")\n>       print(res[0])\nE       KeyError: 0\n\ntest_04_Get_target_device_info.py:29: KeyError"}, "start": 1671699814167, "stop": 1671699814185, "uuid": "0df60a87-e889-4c44-9581-6d67d9d334a7", "historyId": "41c9e7470f0ec327a6f02ec3be0974e5", "testCaseId": "497176acc6aa09da0bda66145c2df40c", "fullName": "Testcase_Devices.test_04_Get_target_device_info.Test_04_Get_target_device_info#test_Get_target_device_info", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "【需求点】：A004-获取固定SN设备的信息"}, {"name": "story", "value": "【正常获取】"}, {"name": "parentSuite", "value": "Testcase_Devices"}, {"name": "suite", "value": "test_04_Get_target_device_info"}, {"name": "subSuite", "value": "Test_04_Get_target_device_info"}, {"name": "host", "value": "SIE180286"}, {"name": "thread", "value": "10960-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcase_Devices.test_04_Get_target_device_info"}]}